general/1_terminal/task4-finiak.txt

```diff
@@ -0,0 +1 @@
+$ cat ./result.json | jq '.messages[].text'
```

Comment:
If you'd like an additional assignemnt, you may try to make the output prettier, getting rid of the arrays in the output.